src/client/components/ResultMessageModalWindow/ResultMessageModalWindow.styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.ResultMessageModalWindow-component {
+  margin-top: 20%;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  border: 1px solid black;
+}
+
+.container-div {
+  position: absolute;
+  width: 100%;
+  padding-top: 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid rgb(202, 15, 15);
+  background-color: white;
+}
+
+.teamAvatar-component img {
+  width: 30%;
+  border-radius: 50%;
+}
+
+.celebration_image-component,
+.teamAvatar-component {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.teamAvatar-component {
+  margin-top: -10%;
+}
+
+.celebration_image-component img {
+  width: 70%;
+  display: flex;
+}
+
+.team_name-component h1 {
+  color: rgb(121, 113, 109);
+  font-family: 'SpicyRice';
```

Comment:
We are not using this font anymore, please check Ghofrane's message on slack.

Reply:
why he doesnt delete it from the font ? .. its still there

Reply:
its also Done :+1: